src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1859,6 +1888,8 @@ void goto_instrument_parse_optionst::help()
     " --value-set-fi-fp-removal    build flow-insensitive value set and replace function pointers by a case statement\n" // NOLINT(*)
     "                              over the possible assignments. If the set of possible assignments is empty the function pointer\n" // NOLINT(*)
     "                              is removed using the standard remove-function-pointers pass. \n" // NOLINT(*)
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --use-rra <file>             apply rr-abstraction to the arrays specified in the json file\n"
```

Comment:
Please would you consider creating a HELP_RRA macro in `rra.h` so that the documentation stays with the code.  It's a  small thing but it means we could add RRA to other tools.